portal/components/react-admin/base/resources/candidateProfile/candidate-show.js

```diff
@@ -0,0 +1,205 @@
+                return getAge({
+                  start: record.DOB,
+                  end: null,
+                }).years;
+              }
+            }}
+          />
+
+          <TextField label="Gender" source="gender.gender_name" />
+          <TextField label="Mobile" source="mobile_number" />
+          <TextField label="Whatsapp" source="whatsapp_mobile_number" />
+          <TextField label="Distict" source="district_name.name" />
+          <TextField label="Pincode" source="pincode" />
+        </Tab>
+        <Tab label="Educational Details">
+          <TextField
+            label="Max Qualification"
+            source="highest_level_qualification.highest_level_qualification_name"
+          />
+          <TextField
+            label="Qualification"
+            source="qualification_detail.qualification_name"
+          />
+          <TextField label="Marks" source="final_score_highest_qualification" />
+        </Tab>
+        <Tab label="Employment History">
+          <TextField
+            label="Are you currently employed?"
+            source="current_employment.current_employment_status"
+          />
+          <TextField
+            label="Have you ever been employed?"
+            source="ever_employment.employment_status"
+          />
+          <FunctionField
+            label="Role and Employer"
+            render={(record) => {
+              if (record) {
+                if (record.current_employed_status === 1) {
+                  return `${record.job_role}, ${record.employer_organization_name}`;
+                } else return "N/A";
+              }
+            }}
+          />
+
+          <FunctionField
+            label="Experience"
+            render={(record) => {
+              if (record && record.work_experience_details) {
+                return `${record.work_experience_details.work_experience_choices}`;
+              } else return "N/A";
+            }}
+          />
+          <FunctionField
+            label="Salary"
+            render={(record) => {
+              if (record) {
+                if (record.current_employed_status === 1) {
+                  return `₹${record.monthly_salary_details.salary_range}`;
+                } else return "N/A";
+              }
+            }}
+          />
+        </Tab>
+        <Tab label="Profile and Preferences">
+          <TextField
+            label="Driving License"
+            source="driver_license.driver_license_choice"
+          />
+          <TextField
+            label="Travel willingness"
+            source="district_travel.district_travel_choice"
+          />
+          <TextField label="PAN CARD" source="pan_card.pan_card_choice" />
+          <TextField
+            label="English Speaking Competency"
+            source="english_knowledge_choice.english_choice"
+          />
+          <TextField
+            label="Computer Skills"
+            source="computer_operator.computer_operator_choice"
```

Comment:
For example, this would be `computer_operator?.computer_operator_choice` and so on for all the `<TextFields>`